### DISCUSSION TITLE
VisualDiff - Thoughts?

### DISCUSSION THREAD

**Discussion by fade2gray** [mmikeww/AHK-v2-script-converter#280] [Show and tell]
I thought I'd try converting VisualDiff to v2, but when I got to `API.hak`, it was like trying to clean up the aftermath of a multi vehicle pile up.

<details>

<summary>So, I did this.</summary>

https://github.com/user-attachments/assets/df5eadd6-e2e8-4ec2-8da3-b7ac495103a1

</details>

**Comment by andymbody** [comment]
I hear there is a v1 to v2 converter available somewhere. Lol.

**Comment by Banaanae** [comment]
There is an issue with regexmatch which causes the function Params to be messed up. I still think ahk-jk is the solution, but anything to get it working

**Comment by Banaanae** [comment]
I created this using a [WebView2 lib](https://github.com/thqby/ahk2_lib/) and [Mergely v5](https://github.com/wickedest/Mergely) (works offline too)
Just need to figure out how to add to the Quick Converter, and maybe do some customising
![image](https://github.com/user-attachments/assets/421e723e-72ad-4939-ab47-bc80901150fe)

**Reply by Banaanae** [reply]
#292
